CodeStarterCamp_Week2/Week2Step3.swift

```diff
@@ -5,22 +5,27 @@
 //  Created by 홍세희 on 2023/08/29.
 //
 
-var allLottoNumberStorage: Dictionary<String , Set<Int>> = [String : Set<Int>]()
+var lottoNumberStorage: [String : Array<Int>] = [:]
 var count: Int = 0
 
 func save(thisWeekLottoNumbers: Set<Int>) {
     count += 1
-    allLottoNumberStorage["\(count)회차"] = thisWeekLottoNumbers
+    let sortedThisWeekLottoNumbers = thisWeekLottoNumbers.sorted()
+    lottoNumberStorage["\(count)회차"] = sortedThisWeekLottoNumbers
 }
 
-//알고 싶은 회차 입력 매개변수 -> "(알고싶은 회차의 숫자)회차"로 입력
-func findWhatLottoNumbers(week: String) {
-    if let _ = allLottoNumberStorage[week] {
-        var numberString: String
-        numberString = (allLottoNumberStorage[week]!).map({ String($0) }).joined(separator: ", ")
-        print(week + "의 로또 당첨 번호는 \(numberString) 입니다. ")
+func findLottoNumbers(weekNumber: Int) {
```

Comment:
파라미터에 전치사를 붙여주고 싶은 마음이 들긴 하네요. 💦
지금도 나쁘지 않습니다!

말씀드렸던 for를 사용하면 호출할 때 자연스럽게 읽힐 것 같습니다.
```suggestion
func findLottoNumbers(forWeek: Int) { // ... }

findLottoNumbers(forWeek: 3)
```